src/docs/admin/references/NotificationServiceReference.adoc

```diff
@@ -295,3 +295,204 @@ and if they have a value depending on the event origin.
 | Y
 | Y
 |=====
+
+[[microsoft_teams_configuration]]
+==== Microsoft Teams notification method
+
+ProActive provides a ready to use Third-Party script that allows users to send notifications to multiple Teams channels.
+
+The script interacts with Microsoft Graph API to:
+
+ - Connect as a user
+ - Send a message to multiple teams channels
+
+Some configuration must be done in ProActive and Microsoft beforehand but it consists in a nutshell of:
+
+- Configuring the notification service to execute the correct script with the correct setup
+- Create or retrieve app information from the Microsoft admin center
```

Comment:
application information ... Microsoft Admin Center

Reply:
App is just the short for Enterprise Application.
Even Microsoft uses app to talk about it.

Example "Integrating an app into your organization's identity system brings challenges in access management, usage evaluation, and reporting. IT Administrators o ...."